src/app/app-routing.module.ts

```diff
@@ -7,8 +7,9 @@ const routes: Routes = [
     loadChildren: () => import('./home/home.module').then((m) => m.HomePageModule),
     canActivate: [AuthGuard],
   },
-  { path: 'list',
-    loadChildren: () => import('./generic-listing-page/generic-listing-page.module').then(m => m.GenericListingPageModule),
+  {
```

Comment:
no changes here right
It shouldnt be pushed
please revert it

Reply:
it is replacing the code block i need to change